crates/swc_ecma_transforms_base/src/helpers/mod.rs

```diff
@@ -393,8 +561,32 @@ let _throw1 = null;
 ",
             false,
             Default::default(),
+            true,
+        )
+    }
+
+    #[test]
+    fn inline_helper_cjs() {
+        crate::tests::test_transform(
+            Default::default(),
+            |_| {
+                enable_helper!(throw);
+                as_folder(InjectHelpers {
+                    module_type: ModuleType::CommonJs,
+                })
+            },
+            "let _throw = null",
+            "function _throw(e) {
+    throw e;
+}
+let _throw1 = null;
+",
+            true, // TODO: we have an extra Stmt in `acutal` AST, can this be fixed?
```

Comment:
@kdy1 is it required to fix this test with `false` passed here? The generated AST has an extra `Stmt` that the expected AST doesn't have.

Reply:
I think it's better to fix it, as it will ensure that the `hygiene` pass does not break injected helpers.